.github/workflows/build.yml

```diff
@@ -22,46 +25,44 @@ jobs:
       DB_USERNAME: postgres
       DB_PASSWORD: postgres
       DB_HOSTNAME: localhost
+      DOCKER_IMAGE: register-trainee-teacher-data
+      DOCKER_REPO: ${{secrets.DOCKERHUB_USERNAME}}/register-trainee-teacher-data
 
     steps:
     - uses: actions/checkout@v2
 
     - name: Get values for current commit
       run: |
         GIT_SHA_SHORT=$(echo ${{github.sha}} | cut -c 1-7)
-        GIT_REF=${{github.ref}}
-        GIT_BRANCH=${GIT_REF##*/}
+        GIT_REF="${{github.ref}}"
```

Comment:
`${{github.ref}}` is a github actions variable. Since you are in the `run`, you can use the shell environment variable`${GITHUB_REF}`

Reply:
done...